devito/passes/clusters/aliases.py

```diff
@@ -1229,18 +1258,34 @@ def filter(self, key):
     def aliaseds(self):
         return flatten(i.aliaseds for i in self._list)
 
+    @property
+    def is_frame(self):
+        """
+        An AliasList is said to be a "frame" if all of its Aliases are frames.
```

Comment:
IF -> IFF?

Reply:
This only checks sufficient (see slack discussion) so would not put IFF